tools/issue-labeler/cmd/compute_new_labels.go

```diff
@@ -45,6 +45,21 @@ func execComputeNewLabels() error {
 	affectedResources := labeler.ExtractAffectedResources(issueBody)
 	labels := labeler.ComputeLabels(affectedResources, regexpLabels)
 
+	// If there are more than 3 service labels, treat this as a cross-provider issue
+	var serviceLabels []string
+	var nonServiceLabels []string
+	for _, l := range labels {
+		if strings.HasPrefix(l, "service/") {
+			serviceLabels = append(serviceLabels, l)
+		} else {
+			nonServiceLabels = append(serviceLabels, l)
```

Comment:
Is this supposed to be 
```suggestion
			nonServiceLabels = append(nonServiceLabels, l)
```
instead? Otherwise we lose track of consecutive non service labels?

Reply:
Yup, thanks for catching this 🤦 . Let me test it a bit more as well.